lib/texture.js

```diff
@@ -854,10 +841,7 @@ module.exports = function createTextureSet (
         target, miplevel, format, info.xOffset, info.yOffset, width, height, 0)
     } else {
       if (!data) {
-        var Dtype = textureDtypes[textureTypesInvert[type]]
-        if (Dtype) {
-          data = new Dtype(width * height * channels)
-        }
+        data = pool.allocType(type, width * height * channels)
```

Comment:
We should really scope this to browsers that throw warning on `null` texture data. How do we do so?

Reply:
Is there any precedent for feature-detection in regl? Ideally I think we'd just feature-detect if it accepts null data since the result probably doesn't change.